SIL.Archiving/Extensions.cs

```diff
@@ -112,13 +112,17 @@ public static void AppendDelimiter(this StringBuilder sb, string value, string d
 		public static string ToLatinOnly(this string s, string charReplacement, string spaceReplacement, string charsToIgnore)
 		{
 			var returnVal = "";
+			int lastDot = s.LastIndexOf(".", StringComparison.InvariantCulture);
 
+			int i = 0;
 			foreach (var c in s)
 			{
 				var test = c.ToString(CultureInfo.InvariantCulture);
 
 				if ((test == charReplacement) || (test == spaceReplacement))  // do not replace the replacement characters
 					returnVal += test;
+				else if (c == '.' && i != lastDot) // only allow one dot in the file name
+					returnVal += spaceReplacement;
```

Comment:
Wouldn't it be better to use `charReplacement` rather than `spaceReplacement` for a period, for consistency with the rest of the logic in this function?